data/json/achievements.json

```diff
@@ -181,5 +181,98 @@
         "description": "Return to the location you started the game"
       }
     ]
+  },
+  {
+    "id": "achievement_cut_1_tree",
+    "type": "achievement",
+    "name": "Timber",
+    "description": "If a tree falls in a forest and no one is around to hear it, does it make a sound?",
+    "requirements": [ { "event_statistic": "num_cuts_tree", "is": ">=", "target": 1, "visible": "when_achievement_completed" } ],
```

Comment:
I think the `visible` entry here is redundant, given that the achievement is self-hiding.